build/openwisp_api/module_settings.py

```diff
@@ -4,6 +4,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'django.contrib.humanize',
```

Comment:
If it's required for radius, do we need it in API container? 🤔

Reply:
I removed it, I think it's not needed, it's needed in the dashboard settings because that container is used for celery, the email is sent in the background.